src/plugins/OauthPlugin/Oauth/Polling.js

```diff
@@ -33,6 +35,15 @@ export default class Polling {
             return;
           }
 
+          // SOP emulation
```

Comment:
and please update text to
`// SOP emulation for E2E tests`
for understand context

Reply:
done